alluxio/alluxio_file_system.py

```diff
@@ -674,3 +674,510 @@ def _validate_path(self, path):
             raise ValueError(
                 "path must be a full path with a protocol (e.g., 'protocol://path')"
             )
+
+
+class AlluxioAsyncFileSystem:
+    """
+    Access Alluxio file system
+
+    Examples
+    --------
+    >>> # Launch Alluxio with ETCD as service discovery
+    >>> alluxio = AlluxioFileSystem(etcd_host=localhost)
+    >>> # Or launch Alluxio with user provided worker list
+    >>> alluxio = AlluxioFileSystem(worker_hosts="host1,host2,host3")
+
+    >>> print(alluxio.listdir("s3://mybucket/mypath/dir"))
```

Comment:
Probably would need to update the comment accordingly.

Reply:
updated